lib/elixir/lib/dict.ex

```diff
@@ -107,6 +107,9 @@ defmodule Dict do
   @type value :: any
   @type t :: list | map
 
+  @type env :: t
+  defmacrocallback new2(env) :: Macro.t
```

Comment:
@josevalim @alco what do you think if we make an `rjust` for `defcallback` output?